src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,20 +6,26 @@ def selection_sort(arr):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc)
-        # Your code here
-
+        for element in range(cur_index + 1, len(arr)):
+            if arr[element] < arr[smallest_index]:
+                smallest_index = element
 
         # TO-DO: swap
-        # Your code here
-
+        arr[smallest_index], arr[cur_index] = arr[cur_index], arr[smallest_index]
+        
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort(arr):
-    # Your code here
-
-
+    # iterate through arr
+    for i in range(0, len(arr)):
+        # inner loop to repeat and decrement len(arr)
+        for j in range(1, len(arr) - i):
+        # if first element > second element
+            if arr[j-1] > arr[j]:
+                # swap
+                arr[j-1], arr[j] = arr[j], arr[j-1]  
     return arr
```

Comment:
Your bubble sort will always run O(n^2).
Since a swap hasn't happened, it means that the array is sorted. Try to optimize the solution.
Having a boolean flag `has_swapped` and changing the outer for loop to a while loop would be a good start.